### PR TITLE
Make ChartMuseum CI version configurable

### LINKED ISSUE #5118: Make ChartMuseum CI version configurable
**Summary**
We are hardcoding the ChartMuseum version we use in CI.

**Background and rationale**
The rest of the versions used along the CI script are properly extracted into separate variables.

**Description**
We should parameterize the ChartMuseum version.

**Acceptance criteria**

- The ChartMuseum version has been parameterized.
- The new variable is documented in the docs.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by antgamdia**
### Description of the change

While working on the e2e tests I noticed we had a hardcoded CharMuseum version, this PR is extracting the version into a separate variable so that we can properly update it.

### Benefits

CM cersion management will be aligned with the rest of the CI component's

### Possible drawbacks

N/A

### Applicable issues

- fixes #5118

### Additional information

N/A

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -27,6 +27,9 @@ parameters:
   OLM_VERSION:
     type: "string"
     default: "v0.21.2"
+  CHARTMUSEUM_VERSION:
+    type: "string"
+    default: "3.9.0"
```
Comment (antgamdia) [inline .circleci/config.yml:32]: I'm also updating from `2.14.2` to `3.9.0` here, but I'll roll the changes back if the PR fails.

Comment (castelblanque) [reply .circleci/config.yml:32]: It seems that it worked 😄

File: script/e2e-test.sh [hunk 2]
```diff
@@ -17,6 +17,7 @@ TEST_TIMEOUT_MINUTES=${5:-"4"}
 DEX_IP=${6:-"172.18.0.2"}
 ADDITIONAL_CLUSTER_IP=${7:-"172.18.0.3"}
 KAPP_CONTROLLER_VERSION=${8:-"v0.32.0"}
+CM_VERSION=${9:-"2.14.2"}
```
Comment (castelblanque) [inline script/e2e-test.sh:20]:
```suggestion
CHARTMUSEUM_VERSION=${9:-"3.9.0"}
```

File: script/e2e-test.sh [hunk 3]
```diff
@@ -38,6 +39,7 @@ fi
 info "Root dir: ${ROOT_DIR}"
 info "Use multicluster+OIDC: ${USE_MULTICLUSTER_OIDC_ENV}"
 info "OLM version: ${OLM_VERSION}"
+info "ChartMuseum version: ${CM_VERSION}"
```
Comment (castelblanque) [inline script/e2e-test.sh:42]:
```suggestion
info "ChartMuseum version: ${CHARTMUSEUM_VERSION}"
```

File: script/e2e-test.sh [hunk 4]
```diff
@@ -324,7 +328,7 @@ fi
 installOrUpgradeKubeapps "${ROOT_DIR}/chart/kubeapps"
 info "Waiting for Kubeapps components to be ready (local chart)..."
 k8s_wait_for_deployment kubeapps kubeapps-ci
-installChartmuseum admin password
+installChartmuseum admin password "${CM_VERSION}"
```
Comment (castelblanque) [inline script/e2e-test.sh:331]:
```suggestion
installChartmuseum admin password "${CHARTMUSEUM_VERSION}"
```

File: script/e2e-test.sh (hunk 2 above)
Comment (castelblanque) [reply script/e2e-test.sh:20]: Suggesting to change the variable name just for future readability.

**Review by castelblanque — APPROVED** [review]

**Review by ppbaena — APPROVED** [review]
LGTM. Nice to add flexibility to select the version.

**Pull request merged**